models/models.js

```diff
@@ -40,6 +41,30 @@ exports.commentsByArticle = (article_id) => {
         })
 }
 
+exports.updateArticle = (article_id, input) => {
+    const alterVotes = Object.values(input)[0]
+    return db.query(`UPDATE articles SET votes = votes + $1 WHERE article_id = $2 RETURNING *;`, [alterVotes, article_id])
+        .then(({ rows }) => {
+            if (!rows.length) {
+                return Promise.reject({ status: 404, msg: 'not found' });
+            }
+            return rows[0]
+        })
+}
+
+
+
+
+// exports.updateTreasure = (treasure_id, input) => {
```

Comment:
Please don't leave in commented out code, ESPECIALLY from a completely different repo.